code42_on_poll_connector.py

```diff
@@ -142,25 +164,25 @@ def _init_container(self, alert_details):
     def _get_container_label(self):
         return self._connector.get_config().get("ingest", {}).get("container_label")
 
-    def _get_file_events(self, param, observations, alert_details):
-        artifact_count = param.get(
-            "artifact_count", DEFAULT_ARTIFACT_COUNT_FOR_POLL_NOW
-        )
+    def _get_file_events(self, observations, alert_details, artifact_count=None):
+        def _have_enough_events():
+            do_limit_count = artifact_count is not None
```

Comment:
Note: Our stylers get mad each other because of PEP `W503 line break before binary operator`...

`black` wanted:

```
return (
                artifact_count is not None
                and is_poll_now
                and len(file_events) >= artifact_count
            )
```

Probably a better way around this